src/options.ts

```diff
@@ -45,14 +45,21 @@ type SpanProcessorFactory = (
 
 type PropagatorFactory = (options: Options) => TextMapPropagator;
 
+export enum TracesExporter {
+  JAEGER = 'jaeger',
```

Comment:
should this be jaeger-thrift or jaeger_thrift?

Reply:
Since this is not in the spec, I'll make that more specific as you suggest using the key from OTel collector: `jaeger-thrift-http`

https://github.com/open-telemetry/opentelemetry-collector/blob/main/examples/k8s/otel-config.yaml